core/src/main/java/org/apache/accumulo/core/metadata/StoredTabletFile.java

```diff
@@ -33,39 +35,47 @@
  * in Upgrader9to10.upgradeRelativePaths()
  */
 public class StoredTabletFile extends TabletFile {
-  private final String metadataEntry;
+  private final TabletFileMetadataEntry metadataEntry;
```

Comment:
Is this variable a duplicate of `TabletFile.normalizedPath`?

Reply:
It's not quite the same, in TabletFile the value is actually generated and created in the constructor and in StoredTabletFile is provided. It's a bit weird with how everything was set up previously and I was trying to stay to the existing pattern but it seems like there could be some refactoring work to be done here as there does seems to be some duplication in some cases.